document/core/text/modules.rst

```diff
@@ -476,7 +484,9 @@ Element segments allow for an optional :ref:`table index <text-tableidx>` to ide
    \begin{array}{llclll}
    \production{element segment} & \Telem_I &::=&
      \text{(}~\text{elem}~~x{:}\Ttableidx_I~~\text{(}~\text{offset}~~e{:}\Texpr_I~\text{)}~~y^\ast{:}\Tvec(\Tfuncidx_I)~\text{)} \\ &&& \qquad
-       \Rightarrow\quad \{ \ETABLE~x, \EOFFSET~e, \EINIT~y^\ast \} \\
+       \Rightarrow\quad \{ \ETABLE~x, \EOFFSET~e, \EINIT~y^\ast \} \\ &&|&
+     \text{(}~\text{elem}~~y^\ast{:}\Tvec(\Tfuncidx_I)~\text{)} \\ &&& \qquad
```

Comment:
Hm, I guess these should now allow optional elemidx binders. However, That requires tweaking the syntax somehow, otherwise it would be syntactically ambiguous, at least in the passive case. The only idea I have right now is putting in a `passive` keyword after the optional elemidx, though that's not particularly compelling.

Once you have that, you'll also need to extend the definition of the idc function to define the initial identifier context in the module section.